rotkehlchen/inquirer.py

```diff
@@ -648,7 +668,24 @@ def _find_usd_price(
                 price, oracle = Inquirer._query_fiat_pair(base=asset, quote=Inquirer.usd)
                 return price, oracle, False
 
-        # continue, asset isn't fiat or a price can be found by one of the oracles (CC for example)
+        # continue, asset isn't fiat, check manual prices
+        if GlobalDBHandler.get_manual_current_price(asset=asset) is not None:
+            price, used_main_currency, _ = Inquirer._try_oracle_price_query(
```

Comment:
Internally `Inquirer._manualcurrent` calls `GlobalDBHandler.get_manual_current_price(asset=asset)`. Should you just query `Inquirer._manualcurrent` and if the prize returned is not zero then return? Otherwise you are repeating a query